js/layout-javascript/simple-list-code.js

```diff
@@ -1205,6 +1205,16 @@ DynamicList.prototype.renderLoopHTML = function (iterateeCb) {
         requestAnimationFrame(render);
       } else{
         _this.$container.find('.simple-list-container').removeClass('loading').addClass('ready');
+
+        $('[data-line-clamp]').each(function() {
```

Comment:
Keep the operation of each LFD instance within itself.

```suggestion
        _this.$container.find('[data-line-clamp]').each(function() {
```